subsys/net/ip/tcp2.c

```diff
@@ -237,16 +237,18 @@ static const char *tcp_th(struct net_pkt *pkt)
 
 	buf[0] = '\0';
 
-	if (th->th_off < 5) {
+	if (th_off(th) < 5) {
 		len += snprintk(buf + len, BUF_SIZE - len,
-				"bogus th_off: %hu", (uint16_t)th->th_off);
+				"bogus th_off: %hu",
+				(uint16_t)th_off(th));
 		goto end;
 	}
 
 	len += snprintk(buf + len, BUF_SIZE - len,
-			"%s Seq=%u", tcp_flags(th->th_flags), th_seq(th));
+			"%s Seq=%u", tcp_flags(th_flags(th)),
+			th_seq(th));
```

Comment:
This one doesn't need moving to next line. There are other places as well, that easily fit into 80 lines.